modules/account-lib/src/coin/dot/transactionBuilder.ts

```diff
@@ -372,6 +388,20 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
 
   // endregion
 
+  /**
+   * Adds a signature to the transaction.
+   *
+   * @param publicKey public key that produced the signature
+   * @param signature raw signature as a hex encoded Buffer
+   */
+  addSignature(publicKey: BasePublicKey, signature: Buffer): void {
```

Comment:
This builder step should be in the base interface

Reply:
Do you mean this?

https://github.com/BitGo/BitGoJS/blob/00cd5662bf9f89c9c4bdab948f6548107c9ef696/modules/account-lib/src/coin/baseCoin/baseTransactionBuilder.ts#L61